lib/net.c

```diff
@@ -1775,6 +1775,27 @@ start_network_stage2(void *arg)
 	return ret;
 }
 
+/* These definitions are meant to support older versions of libcap. */
+#ifndef CAP_AMBIENT_SUPPORTED
+
+#define CAP_AMBIENT_SUPPORTED() (0)
+
+static inline int
+cap_get_ambient(cap_value_t x)
```

Comment:
This causes an unused argument warning, leading to a compilation error due to -Wfatal-errors:
```
gatekeeper/lib/net.c: In function ‘cap_get_ambient’:
gatekeeper/lib/net.c:1784:29: error: unused parameter ‘x’ [-Werror=unused-parameter]
 cap_get_ambient(cap_value_t x)
```